pom.xml

```diff
@@ -3162,7 +3162,7 @@
               <destDir>testapidocs</destDir>
               <name>Test User API</name>
               <description>The HBase Application Programmer's API</description>
```

Comment:
You may also want to change this to Test API

Reply:
Changed to "The HBase Application Programmer's API test code", to align with the description for testdevapidocs.